### PR TITLE
1559: Remove request for regulators to shut down Ethereum

### PR DESCRIPTION
**Pull request by fulldecent**

### CODE REVIEW

File: EIPS/eip-1559.md [hunk 1]
```diff
@@ -323,12 +323,12 @@ This EIP will increase the maximum block size, which could cause problems if min
...
+It is possible that miners will mine empty blocks until such time as the base fee is very low and then proceed to mine half full blocks and revert to sorting transactions by the priority fee.  While this attack is possible, it is not a particularly stable equilibrium as long as mining is decentralized.  Any defector from this strategy will be more profitable than a miner participating in the attack for as long as the attack continues (even after the base fee reached 0).  Since any miner can anonymously defect from a cartel, and there is no way to prove that a particular miner defected, the only feasible way to execute this attack would be to control 50% or more of hashing power.  If an attacker had exactly 50% of hashing power, they would make no Ether from priority fee while defectors would make double the Ether from priority fees.  For an attacker to turn a profit, they need to have some amount over 50% hashing power, which means they can instead execute double spend attacks or simply ignore any other miners which is a far more profitable strategy.
```
Comment (MicahZoltu) [inline EIPS/eip-1559.md:326]: Change: money => Ether

Noting this since GitHub isn't highlighting the diff.

File: EIPS/eip-1559.md [hunk 2]
```diff
@@ -323,12 +323,12 @@ This EIP will increase the maximum block size, which could cause problems if min
...
+By burning the base fee, we can no longer guarantee a fixed token supply.  This could result in economic instability as the long term supply of ETH will no longer be constant over time.  While a valid concern, it is difficult to quantify how much of an impact this will have.  If more is burned on base fee than is generated in mining rewards then ETH will be deflationary and if more is generated in mining rewards than is burned then ETH will be inflationary.  Since we cannot control user demand for block space, we cannot assert at the moment whether ETH will end up inflationary or deflationary, so this change causes the core developers to lose some control over Ethereum's long term token structure.
```
Comment (MicahZoltu) [inline EIPS/eip-1559.md:331]: I don't believe ETH is considered a "token" but rather a "coin".  I believe all native blockchain currencies are "coins" while all non-native assets are "tokens".

Comment (fulldecent) [reply EIPS/eip-1559.md]: Good point. Made some more updates.

**Comment by axic** [comment]
Asking @vbuterin @econoar @AFDudley @mslipper @i-norden @abdelhamidbakhta for review.

**Comment by timbeiko** [comment]
FWIW, I'm against merging this PR as I think the content removed is valuable.

**Review by abdelhamidbakhta — APPROVED** [review]

**Review by abdelhamidbakhta — CHANGES_REQUESTED** [review]
Content removed was useful

**Comment by axic** [comment]
btw, diffs are weird on this PR, but what part do you both refer to?

The diffs only seem to show changing a few terms and not removing anything?

**Comment by abdelhamidbakhta** [comment]
> btw, diffs are weird on this PR, but what part do you both refer to?
> 
> The diffs only seem to show changing a few terms and not removing anything?

Oh yeah you are right. The diffs view tricked me

**Review by abdelhamidbakhta — APPROVED** [review]
Sounds good

**Comment by timbeiko** [comment]
I was fooled by the diff too: I was against removing the entire "Miners Mining Empty Blocks" paragraph, which this actually doesn't do. I'm fine with the terminology update.

**Comment by fulldecent** [comment]
Sorry for the confusion, I'm just learning that GitHub and VS code don't like very long lines in diffs. 

But of course, very long lines is the right way to write Markdown :-)￼

**Comment by MicahZoltu** [comment]
I used to agree on writing markdown like prose text.  Lately, I have come to like one sentence per line, just so diffs work well.  It still allows the paragraph to flow/read reasonably well and when rendered the newlines will be removed, but at the same time you get really clean diffs.

**Pull request merged**